.formatting/Manifest.toml

```diff
@@ -0,0 +1,216 @@
+# This file is machine-generated - editing it directly is not advised
+
+julia_version = "1.8.1"
```

Comment:
This Julia version is not compatible with `Julia=1.9` used in the CI. 

You might use `Pkg.upgrade_manifest()` within a `Julia=1.9`-REPL.

Reply:
As discussed offline, the manifest files will be removed in a separate PR.